parm/config/gfs/config.resources.GAEAC5

```diff
@@ -26,6 +26,8 @@ case ${step} in
 
 esac
 
+export FI_VERBS_PREFER_XRC=0
```

Comment:
Curious what this is for?

Reply:
From Seth Underwood, Gaea SA: "After the C5 update, users reported that some jobs failed during the MPI_Finalize call. We have alerted ORNL and HPE. HPE has suggested setting the environment variable FI_VERBS_PREFER_XRC=0 in the run script (setenv FI_VERBS_PREFER_XRC 0, for csh; export FI_VERBS_PREFER_XRC=0). This has resolved the error in our tests.  Please add this variable to your run script(s) if you also hit this error.  Please note that we do not see any issues preemptively setting this environment variable."  You will also see this setting in the ufs-weather-model now too